consensus/src/simplex/actors/voter/actor.rs

```diff
@@ -2025,7 +2022,18 @@ impl<
         };
     }
 
-    pub async fn run(
+    pub fn start(
+        self,
+        backfiller: resolver::Mailbox,
+        sender: impl Sender,
+        receiver: impl Receiver,
+    ) -> Handle<()> {
+        self.context
```

Comment:
We could add a `spawn_ref` that doesn't consume the context for things like this.

Reply:
Turns out this gets real weird because we maintain a reference in the inner spawn...